### PR TITLE
feat(suite-native): use our custom URL to check network reachability to prevent pinging google

### PR DESCRIPTION
**Pull request by matejkriz**
## Description

No visible changes, only it should now using our CDN for network check.

## Related Issue

Related #13597

### CODE REVIEW

**Review by vytick — APPROVED** [review]
Great that you push this! ❤️ 

LGTM

File: suite-native/app/src/App.tsx [hunk 1]
```diff
@@ -34,6 +34,11 @@ const APP_STARTED_TIMESTAMP = Date.now();
 // Keep the splash screen visible while we fetch resources
 SplashScreen.preventAutoHideAsync();
 
+// Global configuration of NetInfo for network status monitoring.
```
Comment (Nodonisko) [inline suite-native/app/src/App.tsx:37]: Wouldn't be better place for this component in file with function definition?

Comment (matejkriz) [reply suite-native/app/src/App.tsx:37]: You mean for the comment? I was also thinking about that, but I believe it's better to see here why it is being called and what is it about. I don't expect us to expand this in any major way in the future. But both places seem fine to me. @vytick , we need one more vote 😉

Comment (matejkriz) [reply suite-native/app/src/App.tsx:37]: I'll merge as it is, anyone feel free to open PR to enhance the comment 🙏

**Comment by STew790** [comment]
**QA OK**
Works as expected. Correctly shows offline/online status and reloading of graphs or etc works also good.

**Info:**
24.9.1 https://github.com/trezor/trezor-suite/commit/bf893a5e0f152087cb4e47a4921a64c3c4c1624f

**Pull request merged**